src/components/Person/Person.jsx

```diff
@@ -1 +1,17 @@
-// export const Person = ({ person }) => ();
+export const Person = ({
+  person,
+  partner = person.sex === 'm' ? 'wife' : 'husband',
```

Comment:
Avoid using "magic values" (what is m?) and do not complicate the code - move the check and assignment of the value (wife/husband) to the body of the function